docs/contents/components/forms/range-slider/index.ja.mdx

```diff
@@ -272,39 +292,19 @@ return (
 ```tsx
 <VStack gap="lg" my="10">
   <RangeSlider size="sm">
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
-  </RangeSlider>
-
-  <RangeSlider size="md">
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
-  </RangeSlider>
-
-  <RangeSlider size="lg">
-    <RangeSliderMark value={25} w="10" ml="-5">
-      25%
-    </RangeSliderMark>
-    <RangeSliderMark value={50} w="10" ml="-5">
-      50%
-    </RangeSliderMark>
-    <RangeSliderMark value={75} w="10" ml="-5">
-      75%
-    </RangeSliderMark>
+    <For each={["sm", "md", "lg"]}>
```

Comment:
![image](https://github.com/user-attachments/assets/b3bc10cd-f0db-4ff2-8908-f090610768a2)

Reply:
Oops. Thanks for pointing that out!